src/Admin/Views/Providers/Create.cshtml

```diff
@@ -3,14 +3,62 @@
     ViewData["Title"] = "Create Provider";
 }
 
+@section Scripts {
+    <script>
+        function toggleProviderTypeInfo(value) {
+            document.querySelectorAll('[id^="info-"]').forEach(el => { el.classList.add('d-none'); });
+            document.getElementById('info-' + value).classList.remove('d-none');
+        }
+    </script>
+}
+
```

Comment:
You can use `if` statements in cshtml to conditionally display content. See other templates as an example, e.g. https://github.com/bitwarden/server/blob/master/src/Admin/Views/Organizations/Index.cshtml#L13-L21.

So you could use an `if` statement based on the value of `ProviderType`.

Reply:
Yes but this function is to make the UI responsive to the user switching the `ProviderType`. To use the razor `if` statements it would have to postback each time and wouldn't be as responsive.

Reply:
Ah of course. I'm too used to client-side apps 😁 seems like a reasonable solution then.